openmp/runtime/cmake/LibompMicroTests.cmake

```diff
@@ -126,6 +126,7 @@ macro(libomp_test_touch_recipe test_touch_dir)
 endmacro()
 libomp_append(libomp_test_touch_env "KMP_VERSION=1")
 add_custom_target(libomp-test-touch DEPENDS ${libomp_test_touch_targets})
+set_target_properties(libomp-test-touch PROPERTIES FOLDER "OpenMP/Tests")
```

Comment:
I suppose these are for Windows (and potentially Xcode on macOS), especially those "folders" don't look like anything on Linux. I'd recommend `libomo/tests`, `libomp/docs`, etc. We don't call our library "OpenMP" library.

Reply:
* The subproject's directory name is [`openmp`](https://github.com/llvm/llvm-project/tree/main/openmp)
 * The [README ](https://github.com/llvm/llvm-project/blob/89c23f7683efbdf896670218be9095d57ae3e348/openmp/README.rst?plain=1#L2) calls it "LLVM OpenMP Libraries".
 * The website is [*openmp*.llvm.org](https://openmp.llvm.org/), and libomp is not even mentioned. Its title is "LLVM/OpenMP", I think we can drop the "LLVM" prefix within the LLVM repository.
 * `libompd`, `libomptarget` (formerly), `omp.h`, Archer, and ompopt documentation lives in there too, i.e. this is not the subproject for just libomp.

The folder names require an IDE what display them, none of them are available for Linux. They are also not directories in the filesystem, case should not matter.

I could call it "OpenMP Libraries" if that makes it clearer.

Reply:
My point is, when it comes to directory (except the top level folder), we don't call it "OpenMP". I don't have a strong objection, but just find `libomp` more conventional. I'll leave it up to you.